consensus/types/src/validator.rs

```diff
@@ -152,6 +152,17 @@ impl Validator {
             .flatten()
     }
 
+    pub fn get_eth1_withdrawal_credential(&self, spec: &ChainSpec) -> Option<Address> {
+        self.has_eth1_withdrawal_credential(spec)
+            .then(|| {
+                self.withdrawal_credentials
+                    .as_slice()
+                    .get(12..)
+                    .map(Address::from_slice)
+            })
+            .flatten()
+    }
```

Comment:
i hate that we added this function back just for a single place in the spec.. this is gonna cause a bug at some point due to confusing with `get_execution_withdrawal_credential`..

Reply:
I agree, spent quite a bit of time fighting the tests on this one. I'm just gonna use the function directly here instead

Reply:
Fixed in 707338d